app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -1,252 +1,258 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
+    xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
We use 2 spaces for continuation in XML--did the settings in your Android Studio setup diverge by any chance?

Reply:
The reformatting issue has now been fixed. @BenHenning PTAL